cmd/scollector/collectors/ntp_unix.go

```diff
@@ -70,11 +70,9 @@ func c_ntp_peers_unix() (opentsdb.MultiDataPoint, error) {
 		}
 		remote := fields[0]
 		tags := opentsdb.TagSet{"remote": remote, "refid": fields[1]}
-		var current_source int
 		if fl == "*" {
-			current_source = 1
+			tags = tags.Merge(opentsdb.TagSet{"current_source": "1"})
```

Comment:
Conditional tags are generally a bad idea. It will trigger unknowns when you alert on metric{foo=*,current_source=*} when things switch, because that tagset will have "disappeared".

So it would be better to always have the tagkey present and have a value of "true" / "false".

Even with that change, I'm not sure we need an additional metric so would like to see what you are trying to do.

If we do go ahead with make it an additional tag, I would add a new metric most likely so the behavior doesn't change if other people are alerting on the metric.